components/bl-collapsiblePanel-component/styles/index.less

```diff
@@ -24,12 +24,17 @@
 .bl-customComponent-collapsiblePanel {
   width: @bl-customComponent-collapsiblePanel-width;
 
-  &.panel-expanded.with-animation {
+  &.panel-expanded {
     .collapse-icon {
       transform: @bl-customComponent-collapsiblePanel-title-icon-transform;
     }
     .panel-content {
       max-height: @bl-customComponent-collapsiblePanel-content-open-maxHeight;
+    }
+  }
+
+  &.panel-expanded.with-animation {
```

Comment:
Could we move it to block in 27 line?

Reply:
No, we can't, a parameter has been added that is responsible for the expanded / collapsed panel, and if the panel is expanded, then the expansion animation is not applied.